src/test/java/uk/gov/hmcts/juror/api/moj/report/grouped/ReasonableAdjustmentsReportTest.java

```diff
@@ -0,0 +1,159 @@
+package uk.gov.hmcts.juror.api.moj.report.grouped;
+
+import com.querydsl.core.Tuple;
+import com.querydsl.jpa.impl.JPAQuery;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.mockito.MockedStatic;
+import uk.gov.hmcts.juror.api.TestConstants;
+import uk.gov.hmcts.juror.api.juror.domain.CourtLocation;
+import uk.gov.hmcts.juror.api.moj.controller.reports.request.StandardReportRequest;
+import uk.gov.hmcts.juror.api.moj.controller.reports.response.AbstractReportResponse;
+import uk.gov.hmcts.juror.api.moj.controller.reports.response.GroupedTableData;
+import uk.gov.hmcts.juror.api.moj.controller.reports.response.StandardReportResponse;
+import uk.gov.hmcts.juror.api.moj.domain.QJuror;
+import uk.gov.hmcts.juror.api.moj.domain.QJurorPool;
+import uk.gov.hmcts.juror.api.moj.report.AbstractGroupedReportTestSupport;
+import uk.gov.hmcts.juror.api.moj.report.DataType;
+import uk.gov.hmcts.juror.api.moj.report.ReportGroupBy;
+import uk.gov.hmcts.juror.api.moj.repository.PoolRequestRepository;
+import uk.gov.hmcts.juror.api.moj.service.CourtLocationService;
+import uk.gov.hmcts.juror.api.moj.utils.SecurityUtil;
+
+import java.time.LocalDate;
+import java.util.Map;
+
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.mockStatic;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@SuppressWarnings({
+    "PMD.LawOfDemeter",
+    "PMD.ExcessiveImports"
+})
+public class ReasonableAdjustmentsReportTest extends AbstractGroupedReportTestSupport<ReasonableAdjustmentsReport> {
+
+    private MockedStatic<SecurityUtil> securityUtilMockedStatic;
+    private CourtLocationService courtLocationService;
+
+    public ReasonableAdjustmentsReportTest() {
+        super(QJurorPool.jurorPool,
+            ReasonableAdjustmentsReport.RequestValidator.class,
+            ReportGroupBy.builder()
+                .dataType(DataType.COURT_LOCATION_NAME_AND_CODE)
+                .removeGroupByFromResponse(true)
+                .build(),
+            DataType.JUROR_NUMBER,
+            DataType.FIRST_NAME,
+            DataType.LAST_NAME,
+            DataType.POOL_NUMBER_BY_JP,
+            DataType.CONTACT_DETAILS,
+            DataType.NEXT_ATTENDANCE_DATE,
+            DataType.JUROR_REASONABLE_ADJUSTMENT_WITH_MESSAGE);
+    }
+
+    @BeforeEach
+    @Override
+    public void beforeEach() {
+        this.securityUtilMockedStatic = mockStatic(SecurityUtil.class);
+        this.courtLocationService = mock(CourtLocationService.class);
+        this.hasPoolRepository = false;
```

Comment:
This should be setHasPoolRepository(false); and should be apart of the constructor not here